src/Backend/Core/Js/ckeditor/plugins/medialibrary/dialogs/linkDialog.js

```diff
@@ -63,7 +63,17 @@ CKEDITOR.dialog.add(
                                     style: 'margin-top: 20px;'
                                 }
                             ]
-                        }
+                        },
+                        {
+                            type: 'select',
+                            label: jsBackend.locale.msg('EditorSelectInternalPage'),
+                            id: 'localPage',
+                            title: jsBackend.locale.msg('EditorSelectInternalPage'),
+                            items: linkList,
```

Comment:
Is this `linkList` simply a global variable?

Reply:
Seems so... didn't to a thing for that, it was still available from previous version I guess  :)

Reply:
yup, can confirm it's still present.